core/src/solana_utils.rs

```diff
@@ -61,7 +61,7 @@ impl SolanaUtils {
     }
 
     pub async fn poll_slots(
-        rpc_client: Arc<RpcClient>,
```

Comment:
Why did you remove arc?

Reply:
For the lifetime and type of function. It doesn't require the ownership of rpc_client. Therefore the reference of the inner of an Atomic Reference Counter.